WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeOrDiscountLineDetails.swift

```diff
@@ -1,12 +1,12 @@
 import SwiftUI
 
-/// View to add/edit a single fee line in an order, with the option to remove it.
+/// View to add/edit a single fee or discount line in an order, with the option to remove it.
 ///
-struct FeeLineDetails: View {
+struct FeeOrDiscountLineDetails: View {
```

Comment:
Should we add `View` to the name? `FeeOrDiscountLineDetailsView`

Reply:
Done in 224c806c033c35b39da86528b48998e994cf7a21